statsmodels/tsa/tests/test_stattools.py

```diff
@@ -336,6 +337,23 @@ def test_ld(self):
         assert_almost_equal(pacfyw, pacfld, DECIMAL_8)
 
 
+class TestBreakvarHeteroskedasticity(object):
```

Comment:
There is no need to use a class.  This should be a single function.  Classes are mostly vestiges of using nose/unittest.

Reply:
I used a class because I wrote more than one tests for one function.
Here, we can run tests in a class by a single command: 
```console
pytest tests/test_stattools::TestBreakvarHeteroskedasticity
```